docs/gpu.md

```diff
@@ -0,0 +1,74 @@
+# GPU support
+
+```
+kops create cluster gpu.example.com --zones us-east-1c --node-size p2.xlarge --node-count 1 --kubernetes-version 1.6.1
+```
+
+(Note that the p2.xlarge instance type is not cheap, but no GPU instance are)
+
+You can use the experimental hooks feature to install the nvidia drivers:
+
+`> kops edit cluster gpu.example.com`
+```
+spec:
+...
+  hooks:
+  - execContainer:
+      image: kopeio/nvidia-bootstrap:1.6
+```
+
+(TODO: Only on instance groups, or have nvidia-bootstrap detect if GPUs are present..)
+
+In addition, you will likely want to set the `Accelerators=true` feature-flag to kubelet:
+
+`> kops edit cluster gpu.example.com`
+```
+spec:
+...
+  kubelet:
+    featureGates:
+      Accelerators: "true"
+```
+
+`> kops update cluster gpu.example.com --yes`
+
+
+Here is an example pod that runs tensorflow; note that it mounts libcuda from the host:
+
+(TODO: Is there some way to have a well-known volume or similar?)
```

Comment:
TBH I'm not entirely sure.  I do hope that once we get this merged that people that actively use the GPU stuff will contribute improvements.